source/gameengine/Ketsji/KX_LodManager.h

```diff
@@ -57,49 +44,36 @@ class KX_LodList: public CValue
 
 	int m_refcount;
 
-	STR_String m_lodListName;
-
 public:
-	KX_LodList(Object *ob, KX_Scene *scene, KX_BlenderSceneConverter *converter, bool libloading);
-	virtual ~KX_LodList();
-
-	// stuff for cvalue related things
-	virtual CValue *Calc(VALUE_OPERATOR op, CValue *val);
-	virtual CValue *CalcFinal(VALUE_DATA_TYPE dtype, VALUE_OPERATOR op, CValue *val);
-	virtual const STR_String& GetText();
-	virtual double GetNumber();
-	virtual STR_String& GetName();
-	virtual void SetName(const char *name); // Set the name of the value
-	virtual CValue *GetReplica();
+	KX_LodManager(Object *ob, KX_Scene *scene, KX_BlenderSceneConverter *converter, bool libloading);
+	virtual ~KX_LodManager();
 
 #ifdef WITH_PYTHON
 
-	//static PyObject *pyattr_get_(void *self_v, const KX_PYATTRIBUTE_DEF *attrdef);
+	static PyObject *pyattr_get_lodlevels(void *self_v, const KX_PYATTRIBUTE_DEF *attrdef);
 	//static int pyattr_set_(void *self_v, const KX_PYATTRIBUTE_DEF *attrdef, PyObject *value);
 
-	KX_PYMETHOD_DOC(KX_LodList, getLevelMeshName);
-
 #endif //WITH_PYTHON
 
 	/** Get lod level cooresponding to distance and previous level.
 	 * \param scene Scene used to get default hysteresis.
 	 * \param previouslod Previous lod computed by this function before.
 	 * \param distance2 Squared distance object to the camera.
 	 */
-	const KX_LodList::Level& GetLevel(KX_Scene *scene, unsigned short previouslod, float distance2);
+	KX_LodLevel *GetLevel(KX_Scene *scene, unsigned short previouslod, float distance2);
 
 	/// If it returns true, then the lod is useless then.
 	inline bool Empty() const
 	{
 		return m_lodLevelList.empty();
 	}
 
-	KX_LodList *AddRef()
+	KX_LodManager *AddRef()
```

Comment:
Sorry i forgotten that, indeed CValue have already these function in it. Then we can use CValue as base class instead of PyObjectPlus :/